src/misc/inc/StreamUtilities.h

```diff
@@ -0,0 +1,44 @@
+//-----------------------------------------------------------------------bl-
+//--------------------------------------------------------------------------
+//
+// QUESO - a library to support the Quantification of Uncertainty
+// for Estimation, Simulation and Optimization
+//
+// Copyright (C) 2008-2015 The PECOS Development Team
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the Version 2.1 GNU Lesser General
+// Public License as published by the Free Software Foundation.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc. 51 Franklin Street, Fifth Floor,
+// Boston, MA  02110-1301  USA
+//
+//-----------------------------------------------------------------------el-
+
+#ifndef UQ_STREAM_UTILITIES_H
+#define UQ_STREAM_UTILITIES_H
+
+#include <istream>
+
+namespace QUESO
+{
+  class StreamUtilities
+  {
+  public:
+
+    StreamUtilities(){};
+    ~StreamUtilities(){};
+
+    static void skip_comment_lines( std::istream &in, const char comment_start);
```

Comment:
Ah ok, this function doesn't skip *all* comment lines, but stops at the first line that doesn't begin with a comment character.  Would you mind adding a little doxygen comment here?